src/Components/ButtonLogin.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import saveToLocalStorage from '../helpers/index';
+import { redirectAct } from '../Redux/Actions';
+
+const ButtonLogin = (props) => {
```

Comment:
Uma idéia aqui é já fazer o destructuring:
```suggestion
const ButtonLogin = ({ email, password, shouldRedirect }) => {
```